src/storage/src/hummock/store/version.rs

```diff
@@ -639,17 +649,38 @@ impl HummockVersionReader {
                     .collect_vec();
                 fetch_meta_reqs.push((level.level_type, fetch_meta_req));
             } else {
-                let table_infos = prune_ssts(
-                    level.table_infos.iter(),
-                    read_options.table_id,
-                    &table_key_range,
-                );
-                if table_infos.is_empty() {
-                    continue;
+                let mut iters = Vec::new();
```

Comment:
Because the size of meta in overlapping levels is small enough so that we could always preload them in VersionUpdate. And I want to reduce the repeat code in this method....
I always thought the improvement of parallel preload is not clear, so that I think we can remove some code.....

Reply:
Ignoring parallel preload looks okay to me since we don't see clear benefits in benchmark but skipping `prune_sst` means we completely skip key range and table id pruning, which doesn't make sense to me.